tests/package.json

```
@@ -14,7 +14,8 @@
     "test-substrate": "mocha test test/substrate/*.test.*",
     "sim": "mocha test sim/*.sim.*",
     "format": "prettier -w '**/*.ts'",
-    "ts-check": "npx tsc --noEmit"
+    "ts-check": "npx tsc --noEmit",
+    "fetch-fixtures": "sh scripts/fetch-fixtures.sh"
```

lets use bash, I guess sh will just fail for some reason on CI
```suggestion
    "fetch-fixtures": "bash scripts/fetch-fixtures.sh"
```